pyproject.toml

```diff
@@ -46,7 +46,7 @@ default_section = "FIRSTPARTY"
 line_length = 79
 
 [tool.mypy]
-python_version = 3.11
+python_version = 3.10
```

Comment:
would updating the `default_language_version` in the `.pre-commit-config.yaml` to the same value make sense?
note: this change might require updating github workflows